source/documentation/internal/team/ways-of-working.html.erb.md

```diff
@@ -48,6 +48,16 @@ Additionally, demos empower stakeholders with direct insights, enhancing their e
 sessions, we celebrate successes, pinpoint areas for growth, and pivot effectively for upcoming sprints, ensuring our
 continuous development and team cohesion.
 
+### **💡 Milestone Planning**
+
+To enable effective delivery of tackle larger, more complex problems - we dedicate a session to looking ahead and planning what we should work on next and detail the work required.
```

Comment:
I think it should either be:
- To enable delivery of more complex problems
or 
- To enable us to tackle more complex problems